cards_corrections.yml

```diff
@@ -1,4 +1,36 @@
 # Objects to be fixed
+#
+# Format is
+#
+# Card Name:
+#   field to fix: new value
+#
+# Ex:
+#
+# Jace, the Planeswalker:
+#   loyalty: 5
+#   manaCost: 1UUB
+#
+# Fields and their values
+# name: Card Name
+# cmc: 4 (int)
+# colorIdentity:
+#   - U
+#   - B
+# colors:
+#   - Blue
+#   - Black
+# manaCost: 1UUB
+# number: 140
+# rarity: Mythic Rare
+# power: X
+# text: "{5}, {T}: You win the game."
+# toughness: *
+# type: Legendary Artifact Creature - Human Monk
+# url: http://wizards.c0m/link/to/card.png
```

Comment:
Then we remove `cmc` and all other (potentially) generated keys as well?
(e.g. `colorIdentity` and `colors`)

We might want to let people know then that they are generated automatically and shouldn't need corrections.

>"The following fields are generated automatically:
...
If you find mistakes in one of these, please open an issue so we can fix it: repo/issues"